modules/vsphere-problem-detector-metrics.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/using-vsphere-problem-detector-operator.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="vsphere-problem-detector-operator-metrics_{context}"]
+= Metrics for the vSphere Problem Detector Operator
+
+The {operator-name} exposes the following metrics for use by the {product-title} monitoring stack.
+
+.Metrics exposed by the {operator-name}
+[cols="2a,8a",options="header"]
+|===
+|Name |Description
+
+|`vsphere_cluster_check_total`
+|Number of cluster-level checks that were performed by the {operator-name}. This count includes both successes and failures.
+
+|`vsphere_cluster_check_errors`
```

Comment:
The metric name for errors is changed to: `vsphere_cluster_check_errors_total` and `vsphere_node_check_errors_total`.
Detailed info: https://bugzilla.redhat.com/show_bug.cgi?id=1915818

Reply:
Sorry, per the bug: https://bugzilla.redhat.com/show_bug.cgi?id=1916379, the metrics type is changed from counter to gauge, and the naming rules: https://www.robustperception.io/on-the-naming-of-things

Looks like `_total` suffix is no needed.

Reply:
I'll use the names in the vdo repo for commit 89f64e4.  Thank you.